src/rStats.js

```diff
@@ -51,11 +51,19 @@ window.rStats = function rStats ( settings ) {
 
     }
 
+    function getDefaultPathFromScript () {
+        var rStatsScript = Array.from(document.querySelectorAll('script')).filter(function (script) {
+            return /rStats/.test(script.src);
+        })[0];
+        if (!rStatsScript) { return ''; }
+        return rStatsScript.src.split('/').slice(0, -2).join('/') + '/demo/';
```

Comment:
It's a bit weird to go back a directory and into the `demo/` directory in order to get the CSS. Ideally, since the script pulls the CSS automatically, it would be in the `src/` directory instead.